default-rom/default-rom.ino

```diff
@@ -284,11 +288,18 @@ void setup() {
     String wifiLightConf = getConfig("wifilight", "255");
     wifiLight = atoi(wifiLightConf.c_str());
     ennotif = getConfig("ennotif", "on") == "on";
+
+    int waitforwifi=0;
+    while(WiFi.status()!=WL_CONNECTED && waitforwifi++<10) delay(500);
```

Comment:
Yes,there is probably a better way to do what I wanted to do.
Currently, it takes five minutes until the first poll is done. With this change, it is done immediately at startup.